merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/InsertSimulationEventsAction.java

```diff
@@ -50,21 +50,19 @@ private static void batchInsertEventGraph(
       final long datasetId,
       final Duration duration,
       final int transactionIndex,
-      final Timestamp simulationStart,
       final List<Pair<String, Pair<Integer, SerializedValue>>> flattenedEventGraph,
       final PreparedStatement statement
   ) throws SQLException {
-    for (final Pair<String, Pair<Integer, SerializedValue>> entry : flattenedEventGraph) {
+    for (final var entry : flattenedEventGraph) {
       final var causalTime = entry.getLeft();
-      final Pair<Integer, SerializedValue> event = entry.getRight();
+      final var event = entry.getRight();
```

Comment:
Idle question: why are these getting converted from their specific types to `var`s?